.goreleaser.yaml

```
@@ -17,6 +17,7 @@ builds:
     - goos: windows
       goarch: arm64
     main: ./cmd/saturn-l2
+    binary: saturn-L2-node
```

```suggestion
    binary: saturn-l2-node
```

While we're at it, shall we remove the less common upper case letter?

---

Personally, I prefer the lower-case version too. However, when we were renaming repositories, the upper-case version was chosen: https://github.com/filecoin-saturn/L2-node. I am following the same style for consistency.

---

Consistency is very valuable 👍

---

> As a user inspecting Process explorer/Activity monitor, I want to understand which process is the Saturn L2 Node.

:+1: